--- origsrc/pyvorbis-1.4/setup.py	2003-12-19 01:55:56.000000000 -0600
+++ src/pyvorbis-1.4/setup.py	2007-05-17 14:45:19.619813300 -0500
@@ -47,6 +47,7 @@
 
 ogg_include_dir = data['ogg_include_dir']
 ogg_lib_dir = data['ogg_lib_dir']
+ogg_libs = string.split(data['ogg_libs'])
 
 vorbismodule = Extension(name='vorbis',
                          sources=['src/vorbismodule.c',
@@ -61,7 +62,7 @@
                                        ogg_include_dir],
                          library_dirs=[vorbis_lib_dir,
                                        ogg_lib_dir],
-                         libraries=vorbis_libs)
+                         libraries=vorbis_libs + ogg_libs)
 
 setup ( name = "pyvorbis",
         version = pyvorbis_version,
